sqlglot/transforms.py

```diff
@@ -154,8 +154,8 @@ def unnest_to_explode(expression: exp.Expression) -> exp.Expression:
     return expression
 
 
-def explode_to_unnest(expression: exp.Expression) -> exp.Expression:
-    """Convert explode/posexplode into unnest (used in hive -> presto)."""
+def to_unnest_cross_join(expression: exp.Expression) -> exp.Expression:
+    """Convert unnest/explode/posexplode into unnest cross join (Presto requires it to be a cross join)."""
```

Comment:
I changed the name of this since it's no longer explode to unnest, but rather adding the cross join. Advice on naming much appreciated.